docs/src/radiative_equilibrium.md

```diff
@@ -0,0 +1,101 @@
+
+# Example
+A suite of concrete examples are provided here as a guidance for running single column Radiative Equilibrium.  
+
+
+# Radiative Equilibrium
+Radiative Equilibrium can be a useful tool for analyzing atmospheric temperature profiles with differing conditions. To run a radiative equilibrium simulation create a yaml file (or use a default one) and run the following commands in your terminal (after instantiating the necessary packages):
+
+The yaml file should look something like this:
+`mysim.yml`
+```
+rad: "clearsky" 
+idealized_h2o: true 
+hyperdiff: "false" 
+config: "column" 
+initial_condition: "IsothermalProfile" 
+z_max: 70000
+z_elem: 70 
+dz_bottom: 100 
+dz_top: 10000 
+t_end: "654days" 
+dt: "3hours" 
+dt_save_to_sol: "30hours" 
+dt_save_to_disk: "100days" 
+prognostic_surface: "true" 
+job_id: "single_column_radiative_equilibrium_clearsky_prognostic_surface_temp"
+```
+
+Then the command line input should look like this
+```
+julia --project=examples examples/driver.jl --config_file path to mysim.yml
+```
+
+The yaml inputs above can be changed to adjust the kind of simulation desired. Options such as t_end can just be increased to the desired length, other arguments such as radiation type can be changed based on the options in `default_config 2.yml` and `default_config.yml`. Radiative equilibrium is ran with clearsky radiation, a prognostic surface, and generally a single column model.
```

Comment:
```suggestion
The configuration in the yaml file above can be changed to adjust to the kind of simulation desired. Options such as t\_end can be increased to the desired simulation time. Other arguments such as radiation type can be changed based on the options in `default_config 2.yml` and `default_config.yml`.
```

Reply:
(You can see how the docs are rendered by clicking on deployment at the bottom of your PR. What is the default_config 2.yml?)

Reply:
I think default_config 2.yml is like the equivalent of cli_options.jl since the new update? cli_options.jl was taken out so I'm not sure where to point to for the possible simulation arguments?

Reply:
I think all the possible arguments that matter here is in `default_config.yml`